packages/website/public/schema.yml

```diff
@@ -30,6 +30,10 @@ paths:
             schema:
               type: string
               format: binary
+          application/car:
+            schema:
+              type: string
+              format: binary
```

Comment:
Please can you add something to the description to say that CAR files are supported and why?